stdlib/private/TensorFlowUnittest/TensorFlowUnittestRemote.swift

```diff
@@ -78,11 +78,12 @@ private class TestCluster {
     // be started when we set TFE_ContextSetServerDef.
     for i in 1..<taskCount {
       debugLog("Starting task \(i)...")
-      let serverDefText = createServerDef(forTasksConfig: tasks, withTaskIndex: i)
-      let serverDef = TFE_GetServerDef(serverDefText, status)
+      let serverDefText = createServerDef(forTasksConfig: tasks, taskIndex: i)
+      let serverDef: UnsafeMutablePointer! =
+          TFE_GetServerDef(serverDefText, status)
       checkOk(status)
       let server: CTFServer! = TF_NewServer(
-          serverDef!.pointee.data, serverDef!.pointee.length, status)
+          serverDef.pointee.data, serverDef.pointee.length, status)
```

Comment:
```suggestion
        serverDef.pointee.data, serverDef.pointee.length, status)
```
Sorry for another nit: Swift's indentation is always +2, not +4.

Reply:
No worries. Thanks for pointing it out. I was not sure sure about it. :)

Done.